File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int index = 1;
        for(int i = 1; i < nums.length;i++){
            if(nums[i] != nums[i-1]){
                nums[index] = nums[i];
                index++;
            }
        }
        return index;
        // int index = 0;
        // Set<Integer> visited = new HashSet<>();
        // for(int i = 0; i < nums.length;i++){
        //     if(!visited.contains(nums[i])){
        //         visited.add(nums[i]);
        //         nums[index] = nums[i];
        //         index++;
        //     }
        // }
        // return index;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        # first = 1
        # count = 1
        # for idx in range(1,len(nums)):
        #     if nums[idx-1] != nums[idx]:
        #         nums[first] = nums[idx]
        #         first += 1
        # return first
        index = 1
        for idx in range(1, len(nums)):
            if nums[idx-1] != nums[idx]:
                nums[index] = nums[idx]
                index += 1
        return index
        